--- lessphp-0.2.0/plessc~	2010-03-05 10:09:07.000000000 +0200
+++ lessphp-0.2.0/plessc	2011-03-29 21:57:35.707217625 +0300
@@ -1,14 +1,11 @@
-#!/usr/bin/php -q
+#!/usr/bin/php
 <?php
 //
 // command line utility to compile less to stdout
 // leaf corcoran <leafo.net>
 $VERSION = "v0.1.6";
 
-error_reporting(E_ALL);
-$path  = realpath(dirname(__FILE__)).'/';
-
-require $path."lessc.inc.php";
+require_once "lessc.inc.php";
 
 $fa = "Fatal Error: ";
 function err($msg) {
@@ -65,7 +62,7 @@
 }
 
 if (has("v")) {
-	exit($VERSION."\n");
+	die($VERSION."\n");
 }
 
 if (has("r", $loc)) {
@@ -136,6 +133,3 @@
 	err($fa.$ex->getMessage());
 	exit(1);
 }
-
-
-?>
