--- lessphp-0.3.3/plessc~	2012-03-29 19:29:31.000000000 +0300
+++ lessphp-0.3.3/plessc	2012-03-29 19:30:00.205475415 +0300
@@ -1,13 +1,10 @@
-#!/usr/bin/php -q
+#!/usr/bin/php
 <?php
 //
 // command line utility to compile less to stdout
 // leaf corcoran <leafo.net>
 
-error_reporting(E_ALL);
-$path  = realpath(dirname(__FILE__)).'/';
-
-require $path."lessc.inc.php";
+require_once "lessc.inc.php";
 
 $VERSION = lessc::$VERSION;
 
@@ -65,7 +62,7 @@
 }
 
 if (has("v")) {
-	exit($VERSION."\n");
+	die($VERSION."\n");
 }
 
 if (has("r", $loc)) {
@@ -136,6 +133,3 @@
 	err($fa.$ex->getMessage());
 	exit(1);
 }
-
-
-?>
