--- lessphp-0.3.5/plessc~	2012-10-04 06:36:19.000000000 +0300
+++ lessphp-0.3.5/plessc	2012-10-04 06:37:12.239862158 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/php -q
+#!/usr/bin/php
 <?php
 // Command line utility to compile LESS to STDOUT
 // Leaf Corcoran <leafot@gmail.com>, 2012
@@ -38,10 +38,7 @@
 	exit($HELP);
 }
 
-error_reporting(E_ALL);
-$path  = realpath(dirname(__FILE__)).'/';
-
-require $path."lessc.inc.php";
+require_once "lessc.inc.php";
 
 $VERSION = lessc::$VERSION;
 
@@ -213,5 +210,3 @@
 	err($fa.$ex->getMessage());
 	exit(1);
 }
-
-?>
