--- lessphp-0.3.9/./plessc~	2013-11-14 20:36:46.603596704 +0200
+++ lessphp-0.3.9/./plessc	2013-11-14 20:37:09.986558441 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env php
+#!/usr/bin/php
 <?php
 // Command line utility to compile LESS to STDOUT
 // Leaf Corcoran <leafot@gmail.com>, 2012
@@ -38,10 +38,7 @@
 	exit($HELP);
 }
 
-error_reporting(E_ALL);
-$path  = realpath(dirname(__FILE__)).'/';
-
-require $path."lessc.inc.php";
+require_once "lessc.inc.php";
 
 $VERSION = lessc::$VERSION;
 
@@ -213,5 +210,3 @@
 	err($fa.$ex->getMessage());
 	exit(1);
 }
-
-?>
